client/src/utils/validate.js

```diff
@@ -0,0 +1,49 @@
+const valideEmail = (value) => {
+    value = value.trim();
+    const emailRe = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    
+    if (value !== '') {
+        if ( emailRe.test(value) ) {
+            return { valid: true, error: '' };
+        } else {
+            return { valid: false, error: 'wrong email' };
+        }
+    } else {
+        return { valid: false, error: 'enter email' };
+    }
+}
+
+const validePassword = (value) => {
+    const passwordRe = /^(.{8,})$/;
+
+    return passwordRe.test(value) ? { valid: true, error: '' } : { valid: false, error: 'password must be at least 8 characters long' };
+};
+
+const valideName = (value) => {
```

Comment:
maybe rename to `isNameValid`? Usually when method return `bool` method should starts with  `is...`